lib/xboxrt/libc_extensions/direct.c

```diff
@@ -46,26 +46,28 @@ static int convert_error(DWORD winerror)
  */
 
 int _chdir(char* path) {
-    return 0;
+    assert(0);
+    return -1; // Unreachable
```

Comment:
This *is* reachable when disabling assertions.
In that case, `errno` must be set, so I suggest setting it to `EACCES`?

Reply:
Ah crap, forgot you could do that, I'll sort it out